### PR TITLE
Fixes #233: Added Index page for Emoji Heatmapper app

### LINKED ISSUE #233: Add initial UI page for Emoji Heatmapper app
- Issue type: Feature request 

### Short description
As mentioned in PR #230 , create a index.html page, where there are two options, one which redirects to emoji app and the other to emoji-heatmap

### Would you like to work on the issue?
Yes

### PR DESCRIPTION
**Pull request by kavithaenair**
### Short description
Fixes #233 
Added index page for emoji heatmapper app.
Demo link: https://kavithaenair.github.io/emojiHeatmapper/

**Screenshots for the change:**
image

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by kavithaenair** [comment]
@djmgit @singhpratyush @SKrPl @Achint08 Please review.

**Comment by vibhcool** [comment]
This image has scrollbar , maybe it will look better if it fits on screen
screenshot from 2017-07-05 23-33-27

File: emojiHeatmapper/index.html [hunk 1]
```diff
@@ -1,80 +1,66 @@
...
+                the map and also displays the sentiment data.</p>
+            </div>
+            <div class="row">
+                <div class="col-md-3"></div>
+                <div class="col-md-9" style="text-align:center;">
+                    <form class="col-lg-12 hidden-xs ng-pristine ng-valid">
+                        <div class="input-group input-group-lg col-sm-8" style="float:left">
+                            <span class="input-group-btn">
+                            <input type="button" class="btn btn-lg btn-primary" id ="button1"
+                                  value="Emoji Map" onclick="window.location.href='emoji-map.html'">
```
Comment (vibhcool) [inline emojiHeatmapper/index.html]: this one depends on you :) . anchor tag here would be better instead of input tag

**Comment by djmgit** [comment]
@kavithaenair also please rebase your branch

**Comment by kavithaenair** [comment]
@djmgit , yes i will try to implement the switching of apps in the next iteration and will create an issue on this ASAP. 👍   For now am fixing the scrollbar issue and adding nav bar in the apps page.

**Comment by djmgit** [comment]
@kavithaenair cool :+1:

**Comment by kavithaenair** [comment]
@daminisatya @djmgit @vibhcool Please re-review.

**Comment by mariobehling** [comment]
@kavithaenair Please ensure responsiveness.

**Comment by kavithaenair** [comment]
@singhpratyush @mariobehling Made the image responsive. Please review.

**Pull request merged**